leaderboard/service/get_member.go

```diff
@@ -0,0 +1,28 @@
+package service
+
+import (
+	"context"
+
+	"github.com/topfreegames/podium/leaderboard/model"
+)
+
+const getMemberServiceLabel = "get member"
+
+// GetMember return a member info
+func (s *Service) GetMember(ctx context.Context, leaderboard, member string, order string, includeTTL bool) (*model.Member, error) {
+	databaseMembers, err := s.Database.GetMembers(ctx, leaderboard, order, includeTTL, member)
+	if err != nil {
+		return nil, NewGeneralError(getMemberServiceLabel, err.Error())
+	}
+
+	if databaseMembers[0] == nil {
```

Comment:
Is it guaranteed to return at least one member? Won't this cause a panic when no members are returned?